### PR TITLE
Home Page with Header

### PR DESCRIPTION
**Pull request by DimitarIM**
Home Page with the header, has responsiveness issues, and isn't finished as planned with more dynamic animations, links do not work.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,8 +33,26 @@ <h2 class="underline-animation">About us</h2>
...
         </div>
     </header>
-    <div id="bg"></div>
-    
+    <div class="main-container">
+        <div id="bg"></div>
+        <div class="welcome-container">
+            <h2>Welcome</h2>
+            <h3>To</h3>
+            <h1>GitHub</h1>
```
Comment (jasminedns) [inline index.html:41]: I'd use an h2 instead of an h1 and then style it as an h1

File: styles/general.css [hunk 2]
```diff
@@ -6,11 +6,17 @@
...
+}
+html, body {
+    height: 100%;
 }
 
 body {
     background:rgb(26, 34, 48);
     background: linear-gradient(0deg, rgba(26,34,48,1) 0%, rgba(20,19,16,1) 100%);
     background-attachment: fixed;
     font-family:Oswald, Arial;
```
Comment (jasminedns) [inline styles/general.css:19]: Thought we were using Inter as a font

File: index.html (hunk 1 above)
Comment (DimitarIM) [reply index.html:41]: Why though? I don't remember anything about not using it.

File: styles/general.css (hunk 2 above)
Comment (DimitarIM) [reply styles/general.css:19]: Yep the font is just a placeholder. We can always change it if needed

File: index.html (hunk 1 above)
Comment (jasminedns) [reply index.html:41]: He said something about the fact that every H1 should be different in every page, but that was for google stuff so I don't know if it matters now

**Pull request merged**